src/Routes/Groups/Groups.test.js

```diff
@@ -6,6 +6,8 @@ import { render } from '@testing-library/react';
 import { MemoryRouter, Route } from 'react-router-dom';
 import { RegistryContext } from '../../store';
 
+jest.mock('../../utils');
```

Comment:
is this row mandatory?

Reply:
yes, this will do the mock for feature flag call